### PR TITLE
Cross Entropy from logits

### PR DESCRIPTION
**Pull request by msluszniak**
@seanmor5 this is my proposition according to your suggestions.

### CODE REVIEW

**Comment by josevalim** [comment]
:green_heart: :blue_heart: :purple_heart: :yellow_heart: :heart:
